libs/barista-components/secondary-nav/src/section/secondary-nav-group.ts

```diff
@@ -35,7 +39,20 @@ import {
   changeDetection: ChangeDetectionStrategy.OnPush,
   encapsulation: ViewEncapsulation.Emulated,
 })
-export class DtSecondaryNavGroup {
-  /** The text value of the group header */
+export class DtSecondaryNavGroup implements AfterContentInit {
+  /** The text value of the group header. */
   @Input() label: string;
+
+  /** Whether or not the group is active. */
+  active = false;
+
+  /** @internal Active link directive within the group. */
+  @ContentChildren(DtSecondaryNavLinkActive)
+  private _links!: QueryList<DtSecondaryNavLinkActive>;
+
+  ngAfterContentInit(): void {
+    if (this._links.find((link) => link.dtSecondaryNavLinkActive)) {
+      this.active = true;
+    }
+  }
```

Comment:
We may need to think about reacting to changes of the links here, but for now this will be fine. 
I guess there will be a follow up story, where some tests will be added that will trip us up here, if the group does not get destroyed, but the links are dynamically added or removed.